tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1183,6 +1183,87 @@ def _update_iter_stats(self, stats, iter_latency_ms, num_completed_requests,
+        for item in list(self.executor_request_queue.get_request_queue().queue):
+            if not getattr(item, "is_normal_request", False):
+                continue
+            if item.request is None:
+                continue
+            num_queued_context_requests += 1
+            try:
+                num_queued_ctx_tokens += len(item.request.input_token_ids)
+            except Exception:
+                # input_token_ids unavailable for unusual request shapes;
+                # skip the token count for this item.
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)